netconf/rpc.go

```diff
@@ -55,9 +55,10 @@ type RPCReply struct {
 	XMLName   xml.Name   `xml:"rpc-reply"`
 	Errors    []RPCError `xml:"rpc-error,omitempty"`
 	Data      string     `xml:",innerxml"`
-	Ok        bool       `xml:",omitempty"`
+	Ok        bool       `xml:"-"`
 	RawReply  string     `xml:"-"`
 	MessageID string     `xml:"-"`
+	OkPtr     *bool      `xml:"ok,omitempty"`
```

Comment:
This should really be hidden inside a custom UnmarshalXML on the function or a custom type.   Either way OkPtr shouldn't be exposed.

See how i implemented this for v2 (may change):  

https://github.com/nemith/go-netconf/blob/v2/msg.go#L102